content/millennium/dstu2/diagnostic/observation.md

```diff
@@ -44,14 +45,19 @@ _Implementation Notes_
 
 * Currently, the `comments` field may have RTF or other formatted data rather than simple text. This is an issue that will be resolved in a future correction.
 
+* When multiple pages of Observation results are returned for a single query:
+  * All Social history Observations (if any qualify for the query) will be returned on the first page of results. This means that the next bullet does not apply to Social history Observations.
```

Comment:
Feel like this could be somewhat confusing with the next bullet. Considering the part stating "all Observations on the any given page of results are newer..."

Reply:
we'd probably have to separate shx and non-shx into dichotomous sections to rephrase this, is that similar to what you're thinking?

Reply:
I would just say 'This will affect how the results are sorted' or 'If Social history Observations are returned, then they will not be sorted' or 'Social history Observations will not be sorted'

Reply:
Does the _lastUpdated bullet apply for shx?

I was thinking something like:

* When multiple pages of Observation results are returned for a single query:
* If Social history observations are returned, they are on first page
* For subsequent pages containing non-social histories,
** Results are sorted by effective date/time
** _lastUpdated queries are sorted in descending order by last update date time

Reply:
Oh, I see.  The "next bullet point" means only the next bullet point

"Results are sorted by effective date/time in descending order by page. That is, all Observations on any given page of results are newer than all Observations on the next page of results. Sort order within pages is not guaranteed."

If that's the case, maybe we can just reword it a bit.

* All Social history Observations (if any qualify for the query) will be returned on the first page of results and do not follow the sorting rules of subsequent pages.
* All pages not containing social histories are sorted by effective date/time in descending order by page. That is, all Observations on any given page of results are newer than all Observations on the next page of results. Sort order within pages is not guaranteed.
* If the query uses the `_lastUpdated` query parameter, results are sorted by last updated date/time in descending order by page, not by effective date/time.

Reply:
"All pages not containing social histories are sorted by effective date/time in descending order by page. That is, all Observations on any given page of results are newer than all Observations on the next page of results. Sort order within pages is not guaranteed."

Even the pages that do contain SHx obs have newer observations than older pages excepting the SHx themselves. All their non-SHx obs follow the descendin

Reply:
Okay, I see.  I can't think of a better way to word it so I guess we can keep as is.